homeassistant/components/media_player/denon.py

```diff
@@ -101,15 +101,15 @@ def _setup_sources(self, telnet):
     @classmethod
     def telnet_request(cls, telnet, command, all_lines=False):
         """Execute `command` and return the response."""
-        _LOGGER.debug('Sending: "%s"', command)
+        _LOGGER.debug("Sending: %s", command)
```

Comment:
Here changing from '' to "" instead of other way around? It'd however be nice to keep the outputted command quoted as it was.